packages/ui/src/components/notifications/NotificationPosition.ts

```diff
@@ -0,0 +1,6 @@
+export enum NotificationPosition {
+  BottomLeft= 'BottomLeft',
```

Comment:
I do not think we need string values for enums

Reply:
Done